run-tests-with-build-kogito-sources/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>run-tests-with-build-kogito-sources</artifactId>
+
+  <properties>
+    <maven.main.skip>false</maven.main.skip>
+    <maven.test.failure.ignore>false</maven.test.failure.ignore>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>prepare-sources-zip</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <url>${kogito.sources.url}</url>
+          <unpack>true</unpack>
+          <outputDirectory>${sources.directory}</outputDirectory>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <configuration>
+          <!-- kogito-runtimes tests were not designed to run in parallel -->
+          <parallelThreads>1</parallelThreads>
+          <profiles>productized</profiles>
+          <pomIncludes>
+            <pomInclude>kogito-*/*kogito-examples*/pom.xml</pomInclude>
+            <pomInclude>kogito-*/*kogito-runtimes*/pom.xml</pomInclude>
+            <pomInclude>kogito-*/*optaplanner*/pom.xml</pomInclude>
+            <!-- optaweb modules invoked not from top-level to avoid building failing docs modules -->
+            <pomInclude>kogito-*/*optaweb*/*/pom.xml</pomInclude>
+          </pomIncludes>
+          <pomExcludes>
+            <pomExclude>kogito-*/org.kie.rhba-kogito-rhba*/pom.xml</pomExclude>
+            <pomExclude>kogito-*/*optaweb*/*docs/pom.xml</pomExclude>
```

Comment:
This exclude is not needed anymore. The solution is to not skip thee ascidoc. See my commit  https://github.com/kiegroup/run-tests-with-build/pull/22/commits/1078e22985b8290d5c7543ed19b711d4c1d2e316

Also let me know in case you would like to adopt optaweb-parent principle and I can help somehow. @jstastny-cz

Reply:
I am working on that now.